tt_metal/hw/ckernels/grayskull/metal/llk_api/llk_unpack_AB_api.h

```diff
@@ -18,11 +18,13 @@ inline void llk_unpack_AB_hw_configure(
     const uint32_t unpA_operand_id = get_operand_id(unpack_AB_params->unpA_operand);
     const uint32_t unpB_operand_id = get_operand_id(unpack_AB_params->unpB_operand);
 
+    DEBUG_STATUS('U', 'P', 'A', 'B', 'H', 'W');
```

Comment:
one problem...these status chars are restricted to 4 letters - they fit in a 4 byte word. sorry...

Reply:
Fixed it back to 4 characters